menas/src/main/scala/za/co/absa/enceladus/menas/WebJarsResourceFuzzyResolver.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.menas
+
+import org.springframework.web.servlet.resource.WebJarsResourceResolver
+import org.webjars.WebJarAssetLocator
+
+class WebJarsResourceFuzzyResolver(webJarAssetLocator: WebJarAssetLocator)
+  extends WebJarsResourceResolver(webJarAssetLocator) {
+
+  private val webjarPartialPathExtractor = """^.*/([^/]+/[^/]+)/?$""".r
+
+  override def findWebJarResourcePath(path: String): String = {
+    Option(super.findWebJarResourcePath(path))
+      .orElse(findFullPath(path))
+      .flatMap(webjarPartialPathExtractor.findFirstMatchIn(_).map(_.group(1)))
+      .orNull
+
+    val a = Option(super.findWebJarResourcePath(path))
+    val b = a.orElse(findFullPath(path))
+    val c = b.flatMap(webjarPartialPathExtractor.findFirstMatchIn(_).map(_.group(1)))
+    val d = c.orNull
+    d
```

Comment:
This seems to repeat the previous 4 lines

Reply:
Thanks, an artifact of debugging.